api/turing/cluster/servicebuilder/router.go

```diff
@@ -98,7 +98,7 @@ func (sb *clusterSvcBuilder) NewRouterService(
 	// Namespace is the name of the project
 	namespace := GetNamespace(project)
 
-	configMap, err := buildFiberConfigMap(routerVersion, project, experimentConfig)
+	configMap, err := buildFiberConfigMap(routerVersion, project, experimentConfig, envType)
```

Comment:
Why is `envType` being introduced here? Seems like it's not required downstream anyway, since all resources will use the `labeller` package's methods.